src/__testing__/Header.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] All your tests are failing. I think this is because this package was not really installed. Kindly look into that and fix any issues.

  ![image](https://user-images.githubusercontent.com/81468675/206133575-903e0791-608f-45d1-85f6-b5a685210354.png)